CHANGELOG.md

```diff
@@ -1,3 +1,445 @@
+v<2.3.0>, 2020-07-30
+
+PySAL 2.3.0 represents 6 months of enhancements, bug-fixes, widening of test coverage, and improved documentation. All users are encouraged to upgrade to this version as there are numerous optimizations as well as new features (see below) that have been implemented.
```

Comment:
We are leaving the CHANGELOG as `v2.3.0` for now and updating prior to the actual `v2.4.0` release, correct?

Reply:
Correct. This PR is just to get the unit tests dialed in with spopt. Next we move on to the release and change log update in another PR